grpc/service_config/service_config.proto

```diff
@@ -0,0 +1,315 @@
+// Copyright 2016 The gRPC Authors
```

Comment:
Is this the proper date?

Reply:
Yup.  That's when I created the current version of the proto internally.